builder/dockerfile/parser/testfiles/escape-nonewline/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# escape = ``
```

Comment:
It's a bug in the previous implementation that allowed garbage data after the directive https://github.com/docker/docker/blob/61753568f5e482c551df6f3693cb7c942261b7de/builder/dockerfile/parser/parser.go#L51 . Only ` and \ are valid escape characters.